pages/linux/runuser.md

```diff
@@ -0,0 +1,23 @@
+# runuser
+
+> Run commands as a specific user and group without asking for password (needs root privileges).
+
+- Run command as a different user:
+
+`runuser {{user}} -c '{{command}}'`
+
+- Run command as a different user and group:
+
+`runuser {{user}} -g {{group}} -c '{{command}}'`
+
+- Start a login shell as a specific user:
+
+`runuser {{user}} -l`
+
+- Specify a [s]hell for running instead of default shell (also works for login):
```

Comment:
```suggestion
- Specify a shell for running instead of the default shell (also works for login):
```
Sorry missed the article there, also my bad didn't mean to use `[s]`, copy-paste error. :sweat_smile:

Reply:
oh, I think it's format style from our guideline 🙏 ...

Reply:
Btw, it's OK